packages/editor-ui/src/components/CredentialEdit/CredentialEdit.vue

```diff
@@ -264,6 +273,38 @@ export default mixins(showMessage, nodeHelpers).extend({
 			useUsersStore,
 			useWorkflowsStore,
 		),
+		activeNodeType(): INodeTypeDescription | null {
+			const activeNode = this.ndvStore.activeNode;
+
+			if (activeNode) {
+				return this.nodeTypesStore.getNodeType(activeNode.type, activeNode.typeVersion);
+			}
+			return null;
```

Comment:
Early return?

Reply:
This function has come up three times so far I think. Maybe it needs to be shared.